runtime/jvmti/jvmtiHeap10.c

```diff
@@ -568,28 +562,26 @@ wrapHeapIterationCallback(J9JavaVM * vm, J9MM_IterateObjectDescriptor *objectDes
 					iteratorData->userData);
 
 			/* Update the tag only if not autotagging or if it is a class, i.e., if the object is taggable */
-			if ( OBJECT_IS_TAGGABLE(iteratorData->env, vm, object) ) {
```

Comment:
I see use of `autotagging` in some of the comments in this file.  Can you nuke them as well since you're removing autotagging?

Reply:
Done.